File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1151.c
#include <stdio.h>

int main(){
    int i, n, a1, a2, soma, fib[64]={};
    fib[0] = 0;
    fib[1] = 1;
    scanf("%d", &n);
    for(i=2;i<n;i++){
        fib[i]=fib[i-1]+fib[i-2];
    }
    for(i=0;i<n;i++){
        printf("%d", fib[i]);
        if(i==n-1){
            printf("\n");
        } else{
            printf(" ");
        }
    }
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1151.cs
using System;

class URI
{
    static void Main() {
        int i, n, a1, a2, soma;
        int[] fib = new int[64];
        fib[0] = 0;
        fib[1] = 1;
        n = int.Parse(System.Console.ReadLine().Trim());
        for(i=2;i<n;i++){
            fib[i]=fib[i-1]+fib[i-2];
        }
        for(i=0;i<n;i++){
            Console.Write(fib[i]);
            if(i==n-1){
                Console.Write("\n");
            } else{
                Console.Write(" ");
            }
        }
        
    }
}